packages/app-bridge-theme/src/AppBridgeTheme.ts

```diff
@@ -13,8 +13,8 @@ import {
 } from './types';
 
 export interface AppBridgeTheme {
-    dispatch<CommandName extends keyof Command>(
-        dispatchHandler: DispatchHandlerParameter<CommandName, Command>,
+    dispatch<CommandName extends keyof CommandRegistry>(
```

Comment:
IMO it is fine, `dispatch` returns `Promise<void>` we can resolve the promise when we finished the navigation

Reply:
But from a Theme dev experience point of view this could lead to a "not so nice" experience if you always get the warning: `Promise returned from dispatch is ignored`.

Reply:
@mike85 Did you mean something like `public navigate(url: string) {...}` on `AppBridgeTheme`?

Reply:
@Kenny806 I rather meant having sth. generic like the dispatch that does not return a Promise. But I don’t know whether it’s worth it.

Reply:
Ok, let’s merge it. We can still decide afterwards whether it feels good or bad to navigate with the dispatch and the await to prevent having the `Promise returned from dispatch is ignored`.